DigitalLearningSolutions.Web/Views/VerifyEmail/Index.cshtml

```diff
@@ -0,0 +1,130 @@
+          pronoun = "them";
+        }
+        @if (Model.PrimaryEmail != null) {
+          var primaryEmailText = $"Your primary email address{primaryEmail} has not been verified yet." +
+                                 " You can edit your account details, but you cannot access any centre accounts until it is verified.";
+
+          <p class="nhsuk-body-m">@primaryEmailText</p>
+        } else {
+          var (firstCentreName, firstCentreEmail) = Model.CentreSpecificEmails.First();
+          var centreEmailFirstParagraphText = $"Your email address {firstCentreEmail} for your account at" +
+                                              $" {firstCentreName} has not been verified yet. {centreEmailExplanation}";
+          <p class="nhsuk-body-m">@centreEmailFirstParagraphText</p>
+        }
+
+        @foreach (var (centreName, centreEmail) in centreEmailsExcludingFirstParagraph) {
+          var centreEmailText = $"Your email address {centreEmail} for your account at {centreName} has also not been" +
+                                $" verified yet. {centreEmailExplanation}";
+          <p class="nhsuk-body-m">@centreEmailText</p>
+        }
+
+        var verifyEmailExplanation = $"An email with a verification link should have been sent to {emailAddresses}." +
+                                     $" Please click the {link} to verify your email {address}." +
+                                     $" If you have not received {email} after a few minutes, check your Junk {folder}," +
+                                     $" or return to the My account page to resend {pronoun}.";
+
+        <p class="nhsuk-body-m">@verifyEmailExplanation</p>
+      }
+
+      @if (Model.EmailVerificationReason.Equals(EmailVerificationReason.EmailChanged)) {
+        var endingText = Model.UnverifiedEmailsCount == 1
+          ? "If you have not received the email after a few minutes, check your Junk folder," +
+            " or return to the My account page to resend it."
+          : "An email with a verification link has been sent to each of the changed addresses." +
+            " Please click the links to verify your email addresses. If you have not received the emails after a few minutes," +
+            " check your Junk folders, or return to the My account page to resend them.";
+
+        @if (Model.PrimaryEmail != null) {
+          string primaryEmail;
+          string primaryEmailOnlyExplanation;
+          if (Model.UnverifiedEmailsCount == 1) {
+            primaryEmail = null;
+            primaryEmailOnlyExplanation = $"An email with a verification link has been sent to {Model.PrimaryEmail}." +
+                                          " Please click the link to verify your email address. ";
+          } else {
+            primaryEmail = $": {Model.PrimaryEmail}";
+            primaryEmailOnlyExplanation = null;
+          }
+
+          var primaryEmailChangedText = $"You've updated your primary email address{primaryEmail}." +
```

Comment:
Again, I don't see an issue with having the primary email displayed here in all cases. This would get rid of half the logic above, which should then be reduced to a ternary operator.